entrypoint.sh

```diff
@@ -8,10 +8,16 @@ then
 fi 
 VENDOR=$1
 APP_ID=$2
+REPOSITORY_APP_ID=$2
 export KBC_DEVELOPERPORTAL_USERNAME=$3
 export KBC_DEVELOPERPORTAL_PASSWORD=$4
 TAG=$5
 
+if [ $# -gt 5 ]
+then
+  REPOSITORY_APP_ID=$6
```

Comment:
nepovinný parametr, ale když se pošle tak se nastaví u appky repozitář od aplikace jiné... je to třeba v případě když jsou dvě aplikace v dev portálu napojený na jeden image